pipeline/terraform/modules/fargate_service/variables.tf

```diff
@@ -47,13 +47,8 @@ variable "queue_visibility_timeout_seconds" {
 variable "message_retention_seconds" {
   type = number
   # The actual default on SQS is four whole days.
-  #   default = 345600
   # This is sufficient to cope with normal bank holiday weekends.
-
-  # However, if a message is sitting on any main queue for more than a day,
```

Comment:
Was this set up with the assumption that a message would automatically go to the DLQ once its retention period is over?